common/rdr/HexInStream.cxx

```diff
@@ -24,11 +24,10 @@
 #include <rdr/HexInStream.h>
 #include <rdr/Exception.h>
 #include <rfb/util.h>
+#include <algorithm>
```

Comment:
We try to sort system includes before the local ones, and a blank line to separate them. Makes it easier to get an overview of the includes.